openpype/hosts/maya/plugins/publish/extract_playblast.py

```diff
@@ -25,6 +36,31 @@ class ExtractPlayblast(publish.Extractor):
     optional = True
     capture_preset = {}
 
+    def _capture(self, preset, override_viewport_options, instance):
+        filename = preset.get("filename", "%TEMP%")
```

Comment:
Filename isn't used within this method and can just remain in the original process method. Then this method can just return only `path`

Reply:
Yup nice.